check_nscp_api not building on Debian wheezy

refs #5239

```
[ 94%] Building CXX object plugins/CMakeFiles/check_nscp_api.dir/check_nscp_api.cpp.o
cd /home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/obj-x86_64-linux-gnu/plugins && /usr/bin/c++    -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2  -g -pthread -fvisibility=hidden -std=c++0x -O3 -DNDEBUG -I/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2 -I/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/lib -I/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/obj-x86_64-linux-gnu -I/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/obj-x86_64-linux-gnu/lib    -o CMakeFiles/check_nscp_api.dir/check_nscp_api.cpp.o -c /home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp: In function 'void ResultHttpCompletionCallback(const icinga::HttpRequest&, icinga::HttpResponse&, bool&, boost::condition_variable&, boost::mutex&, icinga::Dictionary::Ptr&)':
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:52:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp: In function 'icinga::Dictionary::Ptr QueryEndpoint(const icinga::String&, const icinga::String&, const icinga::String&, const icinga::String&)':
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:92:4: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:109:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp: In function 'int FormatOutput(const Ptr&)':
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:122:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:127:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:131:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:136:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:141:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:149:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:157:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:162:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:174:4: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:178:4: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:221:3: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:225:2: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp: In function 'int main(int, char**)':
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:260:4: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:265:4: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:266:4: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:267:4: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:268:4: error: 'cout' is not a member of 'std'
/home/jenkins/workspace/icinga2-snapshot/deb-debian-wheezy-1binary/arch/x86_64/icinga2/plugins/check_nscp_api.cpp:274:3: error: 'cout' is not a member of 'std'
make[4]: *** [plugins/CMakeFiles/check_nscp_api.dir/check_nscp_api.cpp.o] Error 1
```

https://build.icinga.com/job/icinga2-snapshot/job/deb-debian-wheezy-1binary/arch=x86_64/20/console

--- a/lib/base/process.cpp
+++ b/lib/base/process.cpp
@@ -30,6 +30,7 @@
 #include "base/json.hpp"
 #include <boost/algorithm/string/join.hpp>
 #include <boost/thread/once.hpp>
+#include <iostream>
 
 #ifndef _WIN32
 #	include <execvpe.h>

--- a/plugins/check_nscp_api.cpp
+++ b/plugins/check_nscp_api.cpp
@@ -28,6 +28,7 @@
 #include "base/exception.hpp"
 #include <boost/program_options.hpp>
 #include <boost/algorithm/string/split.hpp>
+#include <iostream>
 
 using namespace icinga;
 namespace po = boost::program_options;

